#!/usr/bin/env python
PACKAGE = "robot_center_mapping"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("lower",		double_t, 0, "lower",    	  -3.0, -10, 10);
gen.add("upper",		double_t, 0, "upper",              -0.2, -10, 10);

gen.add("occupancy_max",		double_t, 0, "occupancy max",    	         -1.2, -10, 10);
gen.add("occupancy_min",		double_t, 0, "occupancy min",   			-2.8, -10, 10);
gen.add("distance",                     double_t, 0, "distance",   		   	    0, -10, 10);

gen.add("weight",		double_t, 0, "weight",    	         3.0,  0, 20);
gen.add("sigma",		double_t, 0, "sigma",                    0.39,  0, 3);
gen.add("radius",		double_t, 0, "radius",                   3.0,  0, 10);
exit(gen.generate(PACKAGE, "robot_center_mapping", "center_mapping"))

